content/docs/09-registries-and-packs/7-advanced-configuration.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Advanced Registry configuration'
```

Comment:
```suggestion
title: 'Advanced Configuration'
```
A bit shorter and to the point. We are already in the context of registries, so it might be a bit redundant.